_test/test_tobyfit/test_let/test_tobyfit_let_resfun_1.m

```diff
@@ -111,7 +111,7 @@ function test_tobyfit_let_resfun_1 (option)
 
 % q-e plot along [0,0,1]
 % ----------------------
-plot(w_nb_qe)
+iplot(w_nb_qe)
```

Comment:
Is `sqw.iplot` a rename of `sqw_old.plot`?

Reply:
Yes